cli/command/service/opts.go

```diff
@@ -101,9 +101,9 @@ func (i *Uint64Opt) Set(s string) error {
 	return err
 }
 
-// Type returns the type of this option
+// Type returns the type of this option, which will be displayed in `--help` output
 func (i *Uint64Opt) Type() string {
-	return "uint64-ptr"
+	return "uint64"
```

Comment:
Yes, we may want to come up with some standard terms if we need to differentiate; suggestions welcome :smile: